common/config/azure-pipelines/templates/publish.yaml

```diff
@@ -6,7 +6,16 @@ parameters:
     default: $(Build.SourceBranchName)
 
 steps:
-  - script: 'node common/scripts/install-run-rush.js publish --apply --publish --include-all --target-branch ${{ parameters.BranchName }} --add-commit-details --set-access-level public'
+  - script: 'node common/scripts/install-run-rush.js publish --apply --publish --pack --include-all --target-branch ${{ parameters.BranchName }} --add-commit-details --set-access-level public --release-folder FOLDER'
     displayName: 'Rush Publish (Policy: ${{ parameters.VersionPolicyName }})'
+
+  - task: PowerShell@2
+    displayName: Publish Packages
+    inputs:
+      targetType: 'filePath'
+      filePath: '$(Build.SourcesDirectory)/common/temp/artifacts/packages'
+      arguments: '-Registry "//onedrive.pkgs.visualstudio.com/_packaging/odsp-npm/npm/registry/"$(DRY_RUN)'
+      workingDirectory: '$(Pipeline.Workspace)/odsp-web-packages/npm'
+      pwsh: true
     env:
-      NPM_AUTH_TOKEN: $(npmToken)
+      SYSTEM_ACCESSTOKEN: $(System.AccessToken)

```

Comment:
```suggestion
```

This gets replaced with an additional `output` item in https://github.com/microsoft/rushstack/blob/main/common/config/azure-pipelines/npm-publish.yaml